esp-hal-common/src/interrupt/riscv.rs

```diff
@@ -349,9 +350,6 @@ mod vectored {
     }
 }
 
-/// # Safety
-///
-/// This function is called from an assembly trap handler.
 #[doc(hidden)]
```

Comment:
Oh, why'd this go away? I thought clippy emits a warning when there's an `unsafe` function has no `Safety` documentation, is that suppressed for `#[doc(hidden)]`?

```suggestion
/// # Safety
///
/// This function is called from an assembly trap handler.
#[doc(hidden)]
```

Reply:
Merging didn't really work out the way i wanted it to so i ended up having to remove some blocks of code from here, i guess i removed this by mistake. Good catch!